sdk/src/wallet/utxo.ts

```diff
@@ -92,14 +110,17 @@ export async function createBitcoinPsbt(
         },
     ];
 
+    if (feeRecipient && feeAmount && feeAmount > DUST_LIMIT) {
+        outputs.push({
+            address: feeRecipient,
+            amount: BigInt(feeAmount),
+        });
+    }
+
     if (opReturnData) {
-        // Strip 0x prefix from opReturn
-        if (opReturnData.startsWith('0x')) {
-            opReturnData = opReturnData.slice(2);
-        }
         outputs.push({
             // OP_RETURN https://github.com/paulmillr/scure-btc-signer/issues/26
-            script: Script.encode(['RETURN', hex.decode(opReturnData)]),
+            script: Script.encode(['RETURN', hex.decode(stripHexPrefix(opReturnData))]),
```

Comment:
Why is this necessary now and wasn't necessary before?

Reply:
It does the same thing as the code I removed above